web/src/components/tasks/anytime-action/tax-clearance-certificate/AnytimeActionTaxClearanceCertificateElement.tsx

```diff
@@ -177,24 +179,18 @@ export const AnytimeActionTaxClearanceCertificateElement = (props: Props): React
                   <Heading level={1}>{props.anytimeAction.name}</Heading>
                 </div>
               </div>
-              <HorizontalStepper
-                steps={stateTaxClearanceCertificateSteps}
-                currentStep={stepIndex}
-                onStepClicked={(step: number): void => {
-                  if (step === 2 && stepIndex === 1) {
-                    saveTaxClearanceCertificateData();
-                  }
-                  setStepIndex(step);
-                }}
-              />
-              {stepIndex === 0 && <TaxClearanceStepOne setStepIndex={setStepIndex} />}
-              {stepIndex === 1 && (
-                <TaxClearanceStepTwo
-                  setStepIndex={setStepIndex}
+              {certificatePdfArray ? (
+                <TaxClearanceDownload />
+              ) : (
+                <TaxClearanceSteps
```

Comment:
I’ve extracted the components that were being rendered as part of the stepper into a separate component. My main goal was to clearly separate the stepper-related logic from the download screen for better organization and maintainability. I chose to leave the download page as-is for now, since including it in the extraction would increase the number of props being passed down, which could add unnecessary complexity. Let me know if you’d prefer that I still pull the download page into the same component.